runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrother.java

```diff
@@ -26,23 +26,26 @@
 
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
+import net.runelite.api.ObjectID;
 import net.runelite.api.Varbits;
 import net.runelite.api.coords.WorldPoint;
 
 @RequiredArgsConstructor
-public enum BarrowsBrothers
+public enum BarrowsBrother
 {
-	AHRIM("Ahrim", new WorldPoint(3566, 3289, 0), Varbits.BARROWS_KILLED_AHRIM),
-	DHAROK("Dharok", new WorldPoint(3575, 3298, 0), Varbits.BARROWS_KILLED_DHAROK),
-	GUTHAN("Guthan", new WorldPoint(3577, 3283, 0), Varbits.BARROWS_KILLED_GUTHAN),
-	KARIL("Karil", new WorldPoint(3566, 3275, 0), Varbits.BARROWS_KILLED_KARIL),
-	TORAG("Torag", new WorldPoint(3553, 3283, 0), Varbits.BARROWS_KILLED_TORAG),
-	VERAC("Verac", new WorldPoint(3557, 3298, 0), Varbits.BARROWS_KILLED_VERAC);
+	AHRIM("Ahrim", new WorldPoint(3566, 3289, 0), Varbits.BARROWS_KILLED_AHRIM, ObjectID.SARCOPHAGUS_20770),
+	DHAROK("Dharok", new WorldPoint(3575, 3298, 0), Varbits.BARROWS_KILLED_DHAROK, ObjectID.SARCOPHAGUS_20720),
+	GUTHAN("Guthan", new WorldPoint(3577, 3283, 0), Varbits.BARROWS_KILLED_GUTHAN, ObjectID.SARCOPHAGUS_20722),
+	KARIL("Karil", new WorldPoint(3566, 3275, 0), Varbits.BARROWS_KILLED_KARIL, ObjectID.SARCOPHAGUS_20771),
+	TORAG("Torag", new WorldPoint(3553, 3283, 0), Varbits.BARROWS_KILLED_TORAG, ObjectID.SARCOPHAGUS_20721),
+	VERAC("Verac", new WorldPoint(3557, 3298, 0), Varbits.BARROWS_KILLED_VERAC, ObjectID.SARCOPHAGUS_20772);
 
 	@Getter
 	private final String name;
 	@Getter
 	private final WorldPoint location;
 	@Getter
 	private final Varbits killedVarbit;
+	@Getter
+	private final int sarcophagus;
```

Comment:
Initialize static final map in this class that has mapping of sarcophagus -> BarrowsBrother, and then add method findByObject(int objectId) to this enum and use that for finding barrows brother from the `menuOptionClicked` method

Reply:
Isn't this a micro optimisation? There's not going to be any speed improvement with an array with the size of 6. It would only make for boilerplate code imo.

Reply:
that is practice what we usually do so its best to follow it.